apps/bare-expo/package.json

```diff
@@ -88,6 +88,7 @@
     "@react-native-community/viewpager": "4.1.6",
     "expo": "~39.0.1",
     "expo-dev-menu": "0.0.2",
+    "expo-dev-menu-interface": "0.0.2",
```

Comment:
That makes me a little bit sad too 😿
Would be great if we could force autolinking to take this dependency automatically. Could you do some research how hard would it be to add sth like this to react native cli?

Reply:
I'll do some research ;)